modules/component/migrations/15_0_0-beta/index.spec.ts

```diff
@@ -0,0 +1,159 @@
+      waitForAsync(async () => {
+        const input = `
+      import { ReactiveComponentModule } from '@ngrx/component';
+
+      @NgModule({
+        imports: [
+          AuthModule,
+          AppRoutingModule,
+          ReactiveComponentModule,
+          CoreModule,
+        ],
+        exports: [ReactiveComponentModule],
+        bootstrap: [AppComponent]
+      })
+      export class AppModule {}
+    `;
+        const expected = `
+      import { LetModule, PushModule } from '@ngrx/component';
+
+      @NgModule({
+        imports: [
+          AuthModule,
+          AppRoutingModule,
+          LetModule, PushModule,
+          CoreModule,
+        ],
+        exports: [LetModule, PushModule],
+        bootstrap: [AppComponent]
+      })
+      export class AppModule {}
+    `;
+
+        appTree.create('./app.module.ts', input);
+        const runner = new SchematicTestRunner('schematics', collectionPath);
+
+        const newTree = await runner
+          .runSchematicAsync(`ngrx-${pkgName}-migration-15-beta`, {}, appTree)
+          .toPromise();
+        const file = newTree.readContent('app.module.ts');
+
+        expect(file).toBe(expected);
+      })
+    );
+    it(
+      `should replace the ReactiveComponentModule in standalone components with LetModule and PushModule`,
+      waitForAsync(async () => {
+        const input = `
+      import { ReactiveComponentModule } from '@ngrx/component';
+
+      @Component({
+        imports: [
+          AuthModule,
+          ReactiveComponentModule
+        ]
+      })
+      export class SomeStandaloneComponent {}
+    `;
+        const expected = `
+      import { LetModule, PushModule } from '@ngrx/component';
+
+      @Component({
+        imports: [
+          AuthModule,
+          LetModule, PushModule
+        ]
+      })
+      export class SomeStandaloneComponent {}
+    `;
+
+        appTree.create('./app.module.ts', input);
+        const runner = new SchematicTestRunner('schematics', collectionPath);
+
+        const newTree = await runner
+          .runSchematicAsync(`ngrx-${pkgName}-migration-15-beta`, {}, appTree)
+          .toPromise();
+        const file = newTree.readContent('app.module.ts');
+
+        expect(file).toBe(expected);
+      })
+    );
+    it(
+      `should not remove the ReactiveComponentModule JS import when used as a type`,
+      waitForAsync(async () => {
+        const input = `
+      import { ReactiveComponentModule } from '@ngrx/component';
+
+      const reactiveComponentModule: ReactiveComponentModule;
+
+      @NgModule({
+        imports: [
+          AuthModule,
+          AppRoutingModule,
+          ReactiveComponentModule,
+          CoreModule
+        ],
+        bootstrap: [AppComponent]
+      })
+      export class AppModule {}
+    `;
+        const expected = `
+      import { ReactiveComponentModule, LetModule, PushModule } from '@ngrx/component';
+
+      const reactiveComponentModule: ReactiveComponentModule;
```

Comment:
The changes in this PR look good to me, but....

Seeing this line of code raised a question.
The `ReactiveComponentModule` is currently deprecated, do we want to remove that in v15?
If that's the case, should we add a comment above this line to mention that ReactiveComponentModule is removed (can be done in a separate PR)?

Reply:
I think we can remove `ReactiveComponentModule` in v15. I also like the idea of adding a comment about its removal 👍

Reply:
I like deleting code ;D. I can open a separate PR.